apps/server/src/controllers/userController.ts

```diff
@@ -104,3 +111,62 @@ export const registerUser = async (req: Request, res: Response) => {
     })
   }
 }
+
+//===================LOGIN-USER========================\\
```

Comment:
No need for this, the user controller handles everything user and the function name matches the comment "login uer"